ReedSolomon 4.0 release fails to compile on windows

```
error: failed to run custom build command for `reed-solomon-erasure v4.0.0`
Caused by:
  process didn't exit successfully: `D:\projects\solana-tjtssd6n5v0uxh9h\target\release\build\reed-solomon-erasure-c3b142865069193e\build-script-build` (exit code: 1)
--- stdout
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CC_x86_64-pc-windows-msvc = None
CC_x86_64_pc_windows_msvc = None
HOST_CC = None
CC = None
CFLAGS_x86_64-pc-windows-msvc = None
CFLAGS_x86_64_pc_windows_msvc = None
HOST_CFLAGS = None
CFLAGS = None
CRATE_CC_NO_DEFAULTS = None
CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
DEBUG = Some("false")
running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.21.27702\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-O2" "-Brepro" "-W4" "-march=native" "-std=c11" "-FoD:\\projects\\solana-tjtssd6n5v0uxh9h\\target\\release\\build\\reed-solomon-erasure-668036bdc653a177\\out\\simd_c\\reedsolomon.o" "-c" "simd_c/reedsolomon.c"
cargo:warning=cl : Command line warning D9002 : ignoring unknown option '-march=native'
cargo:warning=cl : Command line warning D9002 : ignoring unknown option '-std=c11'
reedsolomon.c
D:\.cargo\registry\src\github.com-1ecc6299db9ec823\reed-solomon-erasure-4.0.0\simd_c\reedsolomon.h(24): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory
exit code: 2
```

Maybe turn off simd for Windows?

--- a/core/Cargo.toml
+++ b/core/Cargo.toml
@@ -78,10 +78,10 @@ tokio-io = "0.1"
 untrusted = "0.7.0"
 solana-rayon-threadlimit = { path = "../rayon-threadlimit", version = "0.20.0" }
 
-# reed-solomon-erasure's simd_c feature fails to build for x86_64-pc-windows-msvc, use pure-rust
-[target.'cfg(windows)'.dependencies]
+# reed-solomon-erasure's simd-accel feature fails to build for x86_64-pc-windows-msvc
+[target."cfg(windows)".dependencies]
 reed-solomon-erasure = "4.0"
-[target.'cfg(not(windows))'.dependencies]
+[target."cfg(not(windows))".dependencies]
 reed-solomon-erasure = { version = "4.0", features = ["simd-accel"] }
 
 [dependencies.rocksdb]

